org.spoofax.interpreter.core/src/main/java/org/spoofax/interpreter/stratego/CallT.java

```diff
@@ -107,10 +107,7 @@ public IConstruct eval(final IContext env) throws InterpreterException {
 
         for (int i = 0; i < tvars.length; i++) {
             String formal = formalTermArgs[i];
-            IStrategoTerm actual = tvars[i];
-            // FIXME: This should not be here
```

Comment:
Can you leave this comment in the extracted method? I'd rather preserve the information that there is a hack in there.

Reply:
I added the comment again plus an additional block comment to explain the workaround